lib/openshift/machine_set.rb

```diff
@@ -45,5 +45,11 @@ def is_windows_machinesets?(user: nil, cached: true, quiet: false)
       rr = raw_resource(user: user, cached: cached, quiet: quiet)
       rr.dig('spec', 'template', 'metadata', 'labels','machine.openshift.io/os-id') == "Windows"
     end 
+    def machineset_flavor(user: nil, cached: true, quiet: false)
```

Comment:
We need to defaults quiet to true to show less logs. We still can param it to true when necessary in specific steps.

Reply:
Updated with quiet: true. Thanks!